blackbox/docs/protocols/postgres.txt

```diff
@@ -5,17 +5,15 @@ Postgres Wire Protocol
 
 Crate contains experimental support for the `PostgreSQL wire protocol v3`_.
 
-If a node is started with postgres support enabled it will by default bind to
-``0.0.0.0:5432``. See :doc:`Configuration <../configuration>` for options to
-change that.
+If a node is started with postgres support enabled it will bind to
+``0.0.0.0:5432`` by default. For Crate to use a custom port, the
+setting the corresponding :ref:`conf_ports`
+in the :doc:`Configuration <../configuration>` can be used.
 
-The SQL dialect that Crate supports is different to the SQL dialect that
-PostgreSQL supports. So even though the wire protocol is supported it doesn't
-imply that any postgres client can be used as those clients may use SQL
-statements which aren't supported.
-
-One notable difference is that Crate doesn't support transactions.
-Therefore clients should generally enable ``autocommit``.
+However, even though connecting PostgreSQL tools and client libraries is
+supported, the actual SQL statements have to be supported by Crate's SQL
+dialect. A notable difference there is that Crate doesn't support transactions,
```

Comment:
either: "There is a notable difference..." or: "A notable difference is that ..."